node/bin/src/revm_consistency_checker/bytecode_hash.rs

```diff
@@ -0,0 +1,87 @@
+    let bytecode_hash = calculate_bytecode_hash(&bytecode);
+    assert_eq!(
+        bytecode_hash,
+        B256::from_str("d2efa30ce7bb1a4fd366bfff424b0dd9be445172924b1f7ec0c07a146e58c7b4").unwrap()
+    );
+}
+
+#[test]
+fn calculate_empty_bytecode_test() {
+    use alloy::hex;
+    let bytecode = Bytecode::new_legacy(hex!("").into());
+    let bytecode_hash = calculate_bytecode_hash(&bytecode);
+    assert_eq!(bytecode_hash, EMPTY_BYTE_CODE_HASH);
+}
```

Comment:
Please put tests under `#[cfg(test)] mod tests {}`. Otherwise, AFAIU, `#[test]` would still get expanded and end up in release binary.